packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -92,7 +95,8 @@ def sql_test_connection(self) -> bool:
 
     def sql_columns_metadata(self, table_name: str):
         with self.create_connection().cursor() as cursor:
```

Comment:
Same here, Try / Exception for `create_connection`

We might want to create the connection once in a separate method and return a cursor that can be used in others when necessary?

Reply:
See comment for `sql_tables_metadata`.